doc/development_workflow.md

```diff
@@ -0,0 +1,60 @@
+# Development process in Status Open Bounty
+
+We have a continuously deployed version tracking the `develop` branch live at [test environment](https://openbounty.status.im:444). It uses the [Ropsten](https://ropsten.io/) Ethereum testnet. Any one is welcome to use it and contribute to Open Bounty.
+ 
+Any help is greatly appreciated!
+Currently we use two projects -  `Pipeline For Issues` and `Pipeline For Pull Requests`, issues and pull requests in our repository are passing through all or several stages described below.
+
+## Pipeline For Issues
+
+Issues are added to this project by product owner.
+
+#### To define
+This is backlog for all features/issues/enhancements which we want to include to development process. 
+
+All issues here should be marked with:
+* type - `bug`, `tech-debt`,` enhancement`, `proposal` labels
+* priority - `Prio: high`, `Prio: med`, `Prio: low` labels
```

Comment:
Should we ignore then order of columns and only use these tags for prioritization?

Reply:
I assume order may be relevant if there are issues with same priority?

Reply:
Let it be like this:
`Prio: high`, `Prio: med`, `Prio: low` labels, inside issues with same priority - higher priority in the top. I think we always can reorder issues in our meeting standups or PO can move it to the top if it is crucial for some reason.